### ISSUE TITLE
Release with lsp-types 0.94.0

### ISSUE THREAD

**Issue by denbezrukov** [ebkalderon/tower-lsp#373]
Hello!
Thank you for your library!

`lsp-types` 0.94 provides `position_encodings` in `GeneralClientCapabilities`.

I was wondering if it would be possible to have a  new `tower-lsp` release.

**Comment by ebkalderon** [comment]
Yes! I added support for position encodings to `lsp-types` 0.94 myself ([see upstream PR](https://github.com/gluon-lang/lsp-types/pull/255)), among other things, with the intention of updating `tower-lsp` to support LSP 3.17. A new release of `tower-lsp` is forthcoming; thank you for your patience!

**Comment by denbezrukov** [comment]
Thank you!:)
